### PR TITLE
Enrich some IDs in audit logs with names.

### PR DESCRIPTION
**Pull request by vadimberezniker**
**Related issues**: N/A

### CODE REVIEW

File: enterprise/server/auditlog/auditlog.go [hunk 1]
```diff
@@ -211,6 +197,25 @@ func cleanRequest(e *alpb.Entry_ResourceRequest) *alpb.Entry_ResourceRequest {
...
+		for _, u := range r.Update {
+			uid := u.GetUserId().GetId()
+			userData, err := l.env.GetUserDB().GetUserByID(ctx, uid)
+			if err != nil {
+				return nil, err
+			}
+			if userData.Email != "" {
+				u.UserId.Id += " (" + userData.Email + ")"
+			} else if userData.FirstName != "" && userData.LastName != "" {
+				u.UserId.Id += " (" + userData.FirstName + " " + userData.LastName + ")"
```
Comment (tylerwilliams) [inline enterprise/server/auditlog/auditlog.go:212]: Are you sure you want to do the "(" appending here or is that a client side display thing?

Comment (vadimberezniker) [reply enterprise/server/auditlog/auditlog.go:212]: test

File: enterprise/server/auditlog/auditlog.go [hunk 2]
```diff
@@ -267,6 +272,13 @@ func (l *Logger) GetLogs(ctx context.Context, req *alpb.GetAuditLogsRequest) (*a
 			resourceType = alpb.ResourceType_GROUP
 		}
 
+		er, err := l.enrichRequest(ctx, request)
```
[the remark this answers is not in the document]
Comment (vadimberezniker) [reply enterprise/server/auditlog/auditlog.go]: fixed

**Comment by vadimberezniker** [comment]
Reworked the approach to store ID -> name mappings separately and rendering them in the UI.

**Review by tylerwilliams — APPROVED** [review]

**Pull request merged**